src/main/scala/cromwell/engine/db/slick/ExecutionInfoComponent.scala

```diff
@@ -0,0 +1,49 @@
+package cromwell.engine.db.slick
+
+case class ExecutionInfo(executionId: Int,
+                         key: String,
+                         value: Option[String] = None,
+                         executionInfoId: Option[Int] = None)
+
+case class ExecutionAndExecutionInfo(execution: Execution, executionInfo: ExecutionInfo)
```

Comment:
I think this type could/should go away and its usages replaced by the following type